lib/livechat/index.js

```diff
@@ -63,6 +78,34 @@ LiveChat.prototype.loaded = function(){
 };
 
 /**
+ * Listen for chat events.
+ */
+
+LiveChat.prototype.attachListeners = function(){
+  var self = this;
+  window.LC_API = window.LC_API || {};
+  window.LC_API.on_chat_started = function(data){
+    self.analytics.track('Live Chat Conversation Started',
+      { agentName: data.agent_name },
+      { context: { integration: integration }});
+  };
+  window.LC_API.on_message = function(data){
+    if (data.user_type === 'visitor') {
+      self.analytics.track('Live Chat Message Sent',
+        { messageBody: data.text },
+        { context: { integration: integration }});
+    } else {
+      self.analytics.track('Live Chat Message Received',
+        { messageBody: data.text, agentName: data.agent_name, agentUsername: data.agent_login },
+        { context: { integration: integration }});
+    }
+  };
+  window.LC_API.on_chat_ended = function(){
+    self.analytics.track('Live Chat Conversation Ended');
+  };
```

Comment:
Can you add linebreaks between each of these declarations, just for readability?